util/gem5_bridge/gem5_bridge.c

```diff
@@ -0,0 +1,158 @@
+// SPDX-License-Identifier: BSD-3-Clause OR GPL-2.0-Only
```

Comment:
What's the reason for having two license here? Why GPL-2.0?

We use permissive licensing in gem5. BSD-3-Clause is typically what we use.

Reply:
I believe what Noah has here is correct. We have to have GPL because of the kernel. See https://www.kernel.org/doc/html/v6.13-rc6/process/license-rules.html#id1